scripts/src/javatests/org/oppia/android/scripts/maven/RetrieveLicenseTextsTest.kt

```diff
@@ -45,9 +45,7 @@ class RetrieveLicenseTextsTest {
   private val outContent: ByteArrayOutputStream = ByteArrayOutputStream()
   private val originalOut: PrintStream = System.out
 
-  @Rule
-  @JvmField
-  var tempFolder = TemporaryFolder()
+  @field:[Rule JvmField] val tempFolder = TemporaryFolder()
```

Comment:
TODO: document this & other clean up changes (also explains the todo exemption check updates).

Reply:
Done.